frost-select.md

```diff
@@ -0,0 +1,87 @@
+# ember-frost-select
+
+ * [API](#api)
+ * [Examples](#examples)
+
+## API
+| Attribute       | Type | Value | Description |
+| --------------- | ---- | ----- | ----------- |
+| `data`          | `array` | `[{"label: "foo", "value": "bar"}]` |  An array of "label"/"value" key/value pairs representing the rows in the select drop-down. |
+| `disabled`      | `boolean` | `true`/`false` | disable/enable the component |
+| `error`         | `boolean` | `true`/`false` | tell component to render an error styling |
+| `onChange`     | `string` | `<action-name>` | The action callback to call when the value of the select component changes |
+| `onInput`      | `string` | `<action-name>` | The action callback to call when the value of the filter changes as the user types |
+| `selected`      | `number` or `array` | `1` or `[1, 2]` | The indices of the pre-selected values corresponding to values in the passed-in data. |
+| `selectedValue` | `any`, `array` if using multi-select, `null` to clear | `'bar'` or `['bar', 'buzz']` | A value to choose in the drop down programmatically, or an array of values if using multi-select. Takes precedence over `selected` attribute. Passing `null` will clear the selected state. |
+
+## Examples
+Assuming the following data is available in the consuming context:
+```javascript
+data = [
+  {
+    "label": "foo",
+    "value": "bar"
+  },
+  {
+    "label": "fizz",
+    "value": "buzz"
+  }
+]
+
+singleSelected = 1
+multipleSelected = [1, 2]
+```
+
+### Simple single select
+```
```

Comment:
Add handlebars  ```  "```handlebars" - it will provide the formatted text that will be consistent with other readmes.